lnrpc/routerrpc/config.go

```diff
@@ -25,4 +25,8 @@ type RoutingConfig struct {
 	// AprioriHopProbability is the assumed success probability of a hop in
 	// a route when no other information is available.
 	AprioriHopProbability float64
+
+	// MaxMcHistory defines the maximum number of payment results that
+	// are held on disk by mission control.
+	MaxMcHistory int `long:"maxmchistory" description:"the maximum number of payment results that are held on disk by mission control"`
```

Comment:
Also this struct has many fields that are duplicated below.

Reply:
This is again a consequence of the conditional compilation. Suggestions to remove duplication are welcome.

Reply:
Create a struct with the shared fields, and embed it into both of them.

Reply:
Shared struct created.